test/testsuite/sql/test_description.md

```diff
@@ -0,0 +1,66 @@
+﻿## List of simple tests:
+#### sql_test1
+connect to
+start transaction
+select
+
+#### sql_test2
+include
+declare cursor
+insert into
+savepoint
+open cursor
+fetch cursor
+close
+disconnect
+
+#### sql_test3
+connect
+create table
+declare cursor
+fetch data
+
+Problem: like TSQL009A, there is a Cobol Instruction who disappear between parse and reparse???
```

Comment:
Was this solved by the recent updates by Nicolas ?

Reply:
If yes, please update this comment ;-)